src/Utils/Value.php

```diff
@@ -36,7 +37,7 @@ class Value
     /**
      * Given a type and any value, return a runtime value coerced to match the type.
      *
-     * @param ScalarType|EnumType|InputObjectType|ListOfType|NonNull                              $type
+     * @param ScalarType|EnumType|InputObjectType|ListOfType<Type                                 &InputType>|NonNull $type
```

Comment:
the space is weird

Reply:
https://github.com/webonyx/graphql-php/issues/1020